sdk/test/CrossPlatformTests/CommonTests/Framework/Settings.cs

```diff
@@ -117,7 +117,7 @@ public class StoredSettings
             public string AccountId { get; set; }
             public string AuthProvider { get; set; }
             public string AuthToken { get; set; }
-
+            public string PinpointAppID { get; set; }
```

Comment:
rename this to AppId so that we can reuse it for both Pinpoint's AppId & MobileAnalytics AppId